examples/api/lib/material/menu_bar/menu_bar.0.dart

```diff
@@ -0,0 +1,170 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for MenuBar
+
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() => runApp(const SampleApp());
+
+enum MenuSelection {
+  about('About'),
+  showMessage('ShowMessage'),
+  colorMenu('Color Menu'),
+  colorRed('Red'),
+  colorGreen('Green'),
+  colorBlue('Blue'),
+  quit('Quit');
+
+  const MenuSelection(this.label);
+  final String label;
+}
+
+class SampleApp extends StatelessWidget {
+  const SampleApp({super.key});
+
+  static const String _title = 'MenuBar Sample';
```

Comment:
uber-nit: I would just inline this at the only place where _title is used to make the sample slightly more concise.